app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 17
+    number_of_parts = 4
```

Comment:
No need to have those variables. Just directly pass values
Fix in places below as well